app/controllers/books_controller.rb

```diff
@@ -0,0 +1,8 @@
+class BooksController < ApplicationController
+
+  def show
+    require 'pry'; binding.pry
+    @book = Book.find(params[:id])
+    @review = @book.reviews.find(params[:id])
+  end
```

Comment:
`params[:id]` wouldn't be the id for both a book and a review.

Reply:
From your view you'd have wanted to call @book.reviews and then iterate over the collection to print it to the page.

Reply:
From your view you'd have wanted to call @book.reviews and then iterate over the collection to print it to the page.